nodes/vr_sros/vr-sros.go

```diff
@@ -117,10 +146,13 @@ func (s *vrSROS) CheckInterfaceName() error {
 }
 
 func createVrSROSFiles(node nodes.Node) error {
-	nodes.LoadStartupConfigFileVr(node, configDirName, startupCfgFName)
-
 	nodeCfg := node.Config()
 
+	// use default startup config load function if config in full form is provided
+	if !isPartialConfigFile(nodeCfg.StartupConfig) {
+		nodes.LoadStartupConfigFileVr(node, configDirName, startupCfgFName)
+	}
+
```

Comment:
@steiler does this change look valid to you?

Reply:
yes that was also my first idea until I realized that `LoadStartupConfigFileVr` is creating the node directory in the lab dir.
I did not want to have it created in multiple locations, hence I came up with the stunt as you've seen it.
Maybe we should create the directories as part of `NewDefaultNode(...)`?!?!

Reply:
@steiler I don't see LoadStartupConfigFileVr creating any dirs not related to startup config?